GiniVision/Classes/CameraViewController.swift

```diff
@@ -77,6 +93,7 @@ public typealias CameraErrorBlock = (_ error: CameraError) -> ()
     // Properties
     fileprivate var camera: Camera?
     fileprivate var cameraState = CameraState.notValid
+    fileprivate var filePickerManager: FilePickerManager
```

Comment:
I'm not sure I like the idea of having image pickers in a **Camera** ViewController. I would expect the CameraViewController to only have a camera.

Reply:
Yes, but we cannot change the ViewController name since it would involve more changes for current users. But I think that __Camera__ is referring to the Camera Screen not to the camera itself.